app/controllers/faq_controller.rb

```diff
@@ -4,13 +4,8 @@ class FaqController < ApplicationController
   private
 
   def authenticate_user
-    render plain: "Sorry - This page is only accessible to administrators." if !current_user_is_an_admin?
+    unless user_signed_in? && current_user.admin?
+      render file: Rails.root.join('app', 'views', 'errors', 'unauthorized_user.html.erb'), layout: true, status: :forbidden
```

Comment:
@kholtzman5610 you should just be able to write here something like this
```ruby
render 'errors/unauthorized_user', status: :forbidden
```
but check if I'm wrong